<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductsWindow</class>
 <widget class="QWidget" name="ProductsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1151</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="d_delete">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>20</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer un produit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>130</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saisissez l'id du produit que vous souhaitez modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>190</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher...</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="productId_2">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>150</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QLabel" name="msg_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>230</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>320</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="productsTable_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>661</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="c_modif">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>0</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier un produit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="productId">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>70</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>50</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saisissez l'id du produit que vous souhaitez modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>450</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Catégorie: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>410</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>768</x>
       <y>240</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Description: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>370</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stock: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="description_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>240</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="stock_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>370</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>200</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom: </string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="price_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>410</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="category_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>450</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>190</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modif">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>540</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>100</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>130</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>157</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro QR: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qr_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>150</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="productsTable_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>661</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="supplier_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>490</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>490</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fournisseur :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="b_add">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>-13</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ajouter un produit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>137</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="description">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>187</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="stock">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>317</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="price">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>357</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="categoryComboBox">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>397</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>147</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>718</x>
       <y>187</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Description: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>317</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stock: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>357</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prix: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>397</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Catégorie: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>500</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>97</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro QR: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qr">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>90</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="productsTable_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>661</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>440</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fournisseur :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="supplierComboBox">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>440</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
